### PR TITLE
Add DB Daemon agents for PITR

### PR DESCRIPTION
**Pull request by akinfermo**
PITR = Point-In-Time-Recovery

bug: 209667222
Change-Id: Id266a010c1e52f6be340abf53e20b93e66c06406

### CODE REVIEW

File: oracle/pkg/util/utils.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
+package util
```
Comment (haohu-hh) [inline oracle/pkg/util/utils.go:1]: Can we add license, like other files ?

File: oracle/pkg/util/task/task.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
+// Package task implements tasks struct and its utility methods.
```
Comment (haohu-hh) [inline oracle/pkg/util/task/task.go:1]: Can we add license, like other files ?

File: oracle/pkg/util/secret/secret.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package secret
```
Comment (haohu-hh) [inline oracle/pkg/util/secret/secret.go:1]: Can we add license, like other files ?

Comment (akinfermo) [reply oracle/pkg/util/secret/secret.go:15]: Added. Thanks!

File: oracle/pkg/util/utils.go (hunk 1 above)
Comment (akinfermo) [reply oracle/pkg/util/utils.go:15]: Added. Thanks!

File: oracle/pkg/util/task/task.go (hunk 2 above)
Comment (akinfermo) [reply oracle/pkg/util/task/task.go:15]: Added. Thanks!

**Review by haohu-hh — APPROVED** [review]
LGTM

**Pull request merged**